src/components/CloudProviderAccountOrgIdModal/CloudProviderAccountOrgIdModal.jsx

```
@@ -177,29 +175,42 @@ const CreationTimeInstructions = (props) => {
+        (oldData) => {
+          const result_params = {
+            // @ts-ignore
+            ...oldData?.data?.resourceInstances?.result_params,
+            ...resourceInstance.result_params,
+          };
+
+          return {
+            ...oldData,
+            data: {
+              resourceInstances: [
+                ...(oldData?.data?.resourceInstances || [])?.map((instance) =>
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)